### PR TITLE
EOS-18817-sms2: Fix Firewall Issues for 28100 port

### PR DESCRIPTION
**Pull request by ypise**
Signed-off-by: Yashodhan Pise <yashodhan.pise@seagate.com>

-----
[View rendered docs/User-Guides/LDR-R1 VM Deployment.md](https://github.com/Seagate/cortx-prvsnr/blob/EOS-18817-sms2/docs/User-Guides/LDR-R1 VM Deployment.md)

### CODE REVIEW

**Review by 83bhp — APPROVED** [review]
Looks good, please test before merging!

File: srv/components/system/firewall/config.sls [hunk 1]
```diff
@@ -317,18 +284,20 @@ Management zone:
       - uds
       {% if not 'physical' in grains['virtual'] %}
       - www                 # Open port 80 in management interface for OVA
-      {% endif %}
       {% if salt['cmd.run']('rpm -qa glusterfs-server') %}
       - glusterfs
```
Comment (andkononykhin2) [inline srv/components/system/firewall/config.sls:288]: why that shouldn't be for `physical`?

File: docs/User-Guides/LDR-R1 VM Deployment.md [hunk 2]
```diff
@@ -69,26 +69,27 @@ Checklist:
...
     ```
     [storage_enclosure]
-    type=other
+    type=virtual
 
     [srvnode-1]
     hostname=host1.localdomain
-    network.data_nw.pvt_ip_addr=None
-    network.data_nw.iface=eth1, eth2
+    network.data_nw.pvt_ip_addr=192.254.254.254
```
Comment (sradhanand-pati) [inline docs/User-Guides/LDR-R1 VM Deployment.md:76]: I think this is a typo.
it should be None

Comment (ypise) [reply docs/User-Guides/LDR-R1 VM Deployment.md:76]: This is required for ova.

**Review by sradhanand-pati — APPROVED** [review]

**Pull request merged**